front/src/styles/scss/applications/stdcm/_home.scss

```diff
@@ -107,6 +107,14 @@
 
         .stdcm-vias-list {
           margin-block: 16px;
+
+          button {
+            text-align: left;
+          }
```

Comment:
Could you put that in `_card.scss` ? We will need this for the linked path buttons as well.